exporter/elasticsearchexporter/config_test.go

```diff
@@ -184,7 +184,7 @@ func TestLoadConfig(t *testing.T) {
 					MaxInterval:     1 * time.Minute,
 				},
 				Mapping: MappingsSettings{
-					Mode:  "ecs",
+					Mode:  "raw",
```

Comment:
How about adding another `raw` case instead of change `ecs`?

Reply:
Done in bcdd9b5a36f4276c18b010516e17f3d73deae511.